cedar-policy-validator/src/schema.rs

```diff
@@ -998,7 +998,7 @@ impl ValidatorSchema {
             for p_entity in action.applies_to.applicable_principal_types() {
                 match p_entity {
                     EntityType::Concrete(p_entity) => {
-                        if !entity_types.contains_key(p_entity) {
+                        if !entity_types.contains_key(&p_entity) {
```

Comment:
Clippy and rust-analyzer seem to disagree on if this should have an `&`.

Reply:
maybe the recommendation changed in a recent Rust version?